src/aggregation/agg_result.rs

```diff
@@ -106,6 +112,94 @@ impl From<IntermediateBucketResult> for BucketResult {
                 });
                 BucketResult::Range { buckets }
             }
+            IntermediateBucketResult::Histogram { buckets, req } => {
+                let buckets = if req.min_doc_count() == 0 {
+                    // We need to fill up the buckets for the total ranges, so that there are no
+                    // gaps
+                    let minmax = buckets.iter().minmax_by_key(|bucket| bucket.key);
+                    let all_buckets = match minmax {
+                        itertools::MinMaxResult::MinMax(min, max) => {
```

Comment:
there is a bug if htere is a single bucket.

Reply:
all_buckets is used to fill gaps. In case of a single bucket there is nothing to do, so empty vec is fine